### ISSUE TITLE
Android: Multiple taps could open multiple activity information

### ISSUE THREAD

**Issue by vsabolc** [HTMLCOIN/htmlcoin-android#41]
@senagbe

Steps to reproduce:
- Open the Android app on Samsung S7 
- Log in
- Tap on multiple times in one Transaction
- The Transaction information will be displayed at least two times
- Go back to the Wallet screen
- Tap on one transaction a short after that on the next transaction
- If you are fast enough, both of the transaction information will be displayed

Note: there is no visual problem or overlay, just the activity is loaded more than ones

**Comment by senagbe** [comment]
This is behaviour from the QTUm wallet from which this was forked. Its reasonably low priority though the behaviour doesn't seem ideal.

**Comment by dohaivu** [comment]
I tried many times, I still can't reproduce it

**Comment by vsabolc** [comment]
I agree that this is a low priority issue. It is still reproducible:
https://app.box.com/s/89wzzr6zvkqkn0aifdsimko2labgoa1x

**Comment by senagbe** [comment]
Ok, we'll leave this until there isn't much to do.
